content/collections/docs/new-antlers-parser.md

```diff
@@ -1338,7 +1338,7 @@ The following two examples are functionally equivalent, but each uses a differen
 <a href="{{ $register }}">Register for a new account</a>
 ```
 
-``` antlers
+```antlers
```

Comment:
I think this is a typo and the space needs to be removed.

Reply:
It should work with and without the space.